src/nls/ko/strings.js

```diff
@@ -92,26 +92,26 @@ define({
 
     // Application error strings
     "ERROR_IN_BROWSER_TITLE": "{APP_NAME} 아직 브라우저에서 실행되지 않습니다.",
-    "ERROR_IN_BROWSER": "{APP_NAME}는 HTML로 구축되어 있지만 데스크톱 응용 프로그램으로 실행하여 로컬 파일을 편집 할 수 있습니다. {APP_NAME}을 실행하기 위해 <b>github.com / adobe / brackets-shell</b> 저장소 응용 프로그램 셸을 사용하십시오.",
+    "ERROR_IN_BROWSER": "{APP_NAME}은(는) HTML로 구축되어 있지만 데스크톱 응용 프로그램으로 실행하여 로컬 파일을 편집 할 수 있습니다. {APP_NAME}을 실행하기 위해 <b>github.com / adobe / brackets-shell</b> 저장소 응용 프로그램 셸을 사용하십시오.",
     
     // ProjectManager max files error string
-    "ERROR_MAX_FILES_TITLE": "파일을 인덱스 하는 동안 에러가 발생했습니다.",
-    "ERROR_MAX_FILES": "인덱싱 할 수있는 파일의 최대 수에 도달했습니다. 인덱스에서 파일 검색 기능이 제대로 작동하지 않을 수 있습니다.",
+    "ERROR_MAX_FILES_TITLE": "파일을 인덱스 하던 중에 에러가 발생했습니다.",
+    "ERROR_MAX_FILES": "인덱싱 할 수 있는 파일의 최대 수에 도달했습니다. 인덱스에서 파일 검색 기능이 제대로 작동하지 않을 수 있습니다.",
 
     // Live Development error strings
     "ERROR_LAUNCHING_BROWSER_TITLE": "브라우저 시작 에러",
     "ERROR_CANT_FIND_CHROME": "Google Chrome 브라우저를 찾을 수 없습니다. 설치되어 있는지 확인하십시오.",
-    "ERROR_LAUNCHING_BROWSER": "브라우저를 시작할 때 에러가 발생했습니다. (에러 {0})",
+    "ERROR_LAUNCHING_BROWSER": "브라우저를 시작하던 중에 에러가 발생했습니다. (에러 {0})",
     
     "LIVE_DEVELOPMENT_ERROR_TITLE": "실시간 미리보기 에러",
     "LIVE_DEVELOPMENT_RELAUNCH_TITLE": "브라우저에 연결합니다",
-    "LIVE_DEVELOPMENT_ERROR_MESSAGE": "실시간 미리보기에 연결하려면 원격 디버깅을 사용하여 Chrome을 다시 시작해야합니다. <br /> Chrome을 다시 시작하고 원격 디버깅을 사용 하시겠습니까?",
+    "LIVE_DEVELOPMENT_ERROR_MESSAGE": "실시간 미리보기에 연결하려면 원격 디버깅을 사용하여 Chrome을 다시 시작해야 합니다. <br /> Chrome을 다시 시작하고 원격 디버깅을 사용 하시겠습니까?",
     "LIVE_DEV_LOADING_ERROR_MESSAGE": "실시간 미리보기 페이지를 읽을 수 없습니다.",
-    "LIVE_DEV_NEED_HTML_MESSAGE": "실시간 미리보기를 시작하려면 HTML 파일을 열거 나 index.html 파일이 프로젝트에 포함되어 있는지 확인하십시오.",
-    "LIVE_DEV_NEED_BASEURL_MESSAGE": "서버 측 파일 실시간 미리보기를 시작하려면이 프로젝트의 기본 URL을 지정해야합니다.",
-    "LIVE_DEV_SERVER_NOT_READY_MESSAGE": "실시간 미리보기에 사용되는 HTTP 서버를 시작할 때 에러가 발생했습니다. 잠시 후 다시 시도하세요.",
+    "LIVE_DEV_NEED_HTML_MESSAGE": "실시간 미리보기를 시작하려면 HTML 파일을 열거나 index.html 파일이 프로젝트에 포함되어 있는지 확인하십시오.",
+    "LIVE_DEV_NEED_BASEURL_MESSAGE": "서버 측 파일 실시간 미리보기를 시작하려면 이 프로젝트의 기본 URL을 지정해야 합니다.",
+    "LIVE_DEV_SERVER_NOT_READY_MESSAGE": "실시간 미리보기에 사용되는 HTTP 서버를 시작할 때 에러가 발생했습니다. 잠시 후에 다시 시도하세요.",
     "LIVE_DEVELOPMENT_INFO_TITLE": "실시간 미리보기에 오신 것을 환영합니다",
-    "LIVE_DEVELOPMENT_INFO_MESSAGE": "실시간 미리보기에 의해 {APP_NAME}가 브라우저에 연결했습니다. 브라우저에서 HTML 파일이 수정되면 미리보기가 즉시 업데이트됩니다. <br />이 초기 버전의 {APP_NAME}는 <strong> Google Chrome </strong>에서만 실시간 미리보기 기능하고 <strong> CSS 또는 HTML 파일 </strong>의 편집시에만 라이브 업데이트가 이루어집니다. JavaScript 파일에 대한 변경 사항은 저장시 자동으로 다시로드됩니다. <br /> (이 메시지는 한 번만 표시되지 않습니다.)",
+    "LIVE_DEVELOPMENT_INFO_MESSAGE": "실시간 미리보기에 의해 {APP_NAME}(가) 브라우저에 연결했습니다. 브라우저에서 HTML 파일이 수정되면 미리보기가 즉시 업데이트됩니다. <br />이 초기 버전의 {APP_NAME}은(는) <strong> Google Chrome </strong>에서만 실시간 미리보기 기능하고 <strong> CSS 또는 HTML 파일 </strong>의 편집시에만 라이브 업데이트가 이루어집니다. JavaScript 파일에 대한 변경 사항은 저장시 자동으로 다시로드됩니다. <br /> (이 메시지는 한 번만 표시되지 않습니다.)",
```

Comment:
Because {APP_NAME} is always "Brackets", you don't need to use both two postpositions. The last korean sentence is mistranslated. Its corresponding english sentence is "You'll only see this message once", so it should say "이 메시지는 한 번만 표시됩니다". Here is my suggestion:

"{APP_NAME}가 실시간 미리보기를 통해 브라우저에 연결됐습니다. HTML 파일이 수정되면 브라우저에서 미리보기가 즉시 업데이트됩니다.<br /><br />현재 사용 중인 {APP_NAME} 초기 버전에서는 <strong>Google Chrome </strong>에서만 실시간 미리보기 기능하고 <strong>CSS 또는 HTML 파일</strong>을 편집할 때만 실시간으로 업데이트됩니다. 자바스크립트 파일에 대한 변경 사항은 저장할 때 자동으로 다시 로드됩니다.<br />(이 메시지는 한 번만 표시됩니다)"